### ISSUE TITLE
False positive for Android Chrome 81

### ISSUE THREAD

**Issue by pheinicke** [amilajack/eslint-plugin-compat#307]
> URL is not supported in Android Chrome 81

According to caniuse.com it is supported: https://caniuse.com/#feat=url

I am using 
`"eslint-plugin-compat": "^3.5.1"`

```
"browserslist": [
    "and_chr >= 78",
    "ios_saf >= 11"
]
```

**Comment by nasatome** [comment]
Similar error: 
ESLint: document.querySelector() is not supported in Android Browser 80, Android Chrome 80(compat/compat)

**Comment by amilajack** [comment]
Try using `eslint-plugin-compat@next` and let me know if it works for you. It comes with some bug fixes to compat resolution.

**Comment by nasatome** [comment]
it's okay now

![image](https://user-images.githubusercontent.com/18271791/81754093-e8bf6600-947a-11ea-9e5c-c9623b9d0bd8.png)

**Comment by amilajack** [comment]
Just pushed the `@next` tag to `eslint-plugin-compat@3.6.0`. You can use that version instead now. Thanks for confirming this works.
